lua/telescope/pickers/_test.lua

```diff
@@ -1,244 +0,0 @@
-local assert = require "luassert"
```

Comment:
Is this file delete intentionally or is it debug the ci?

Reply:
I deleted this because the only usage of it that I found was [in the scrolling specs](https://github.com/nvim-telescope/telescope.nvim/pull/2287/files#diff-e28029904538b8499b83db28a26c7110e4a85cd6f618fd88ac1a11b3feca047bL3), which seems to timeout on other pull requests too. 

- [This is a recent failing run from master](https://github.com/nvim-telescope/telescope.nvim/actions/runs/3855177396/jobs/6569957658#step:6:287). 
- [These are the usages of it in the current master](https://github.com/search?q=repo%3Anvim-telescope%2Ftelescope.nvim+require+%22telescope.pickers._test%22&type=code).

As far as I see the `require("telescope.testharness").run_file` creates a process similarly like this one used to, but this one failed to get the response from the scrolling test for some reason (I haven't looked into it deeper).

Reply:
cc @Conni2461 what do you think?

If it's okay with @Conni2461, this file should be deleted in a different PR called something like fix(tests) or ref(test): remove packers/_test.lua

Reply:
this was the old testing framework and should actually be removed. Separate PR is good idea. Thanks :)

Reply:
https://github.com/nvim-telescope/telescope.nvim/pull/2307